cli/node/analyze.rs

```diff
@@ -67,11 +67,11 @@ pub fn esm_code_with_node_globals(
   let global_this_expr = if has_global_this {
     global_this_expr
   } else {
-    write!(result, "var globalThis = {};", global_this_expr).unwrap();
+    write!(result, "var globalThis = {};\n", global_this_expr).unwrap();
```

Comment:
I think we want to keep this all on one line because then it only affects the first lines column count (same with below). We should add a comment here saying that.

Reply:
I'll revert it before merging, just added it to make debugging easier